Instagram_App/Instagram_App/Controllers/HouseViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ViewController.swift
+//  Instagram_App
+//
+//  Created by Анастасия Козлова on 17.10.2022.
+//
+
+import UIKit
+
+/// Главная страница - лента инстаграм
+final class HouseViewController: UIViewController {
+    // MARK: - Private VConstant
+    private enum Constant {
+        static let indentifireStoryText = "story"
+        static let indentifirePostText = "post"
+        static let indentifireRecText = "rec"
+    }
+
+    // MARK: - IBOutlet
+    @IBOutlet private weak var instTableView: UITableView!
+
+    // MARK: - Private Visual Components
+    private let refreshControl: UIRefreshControl = {
```

Comment:
```suggestion
    private let refreshControl = {
```
Swift же умеет в неявное указание типа, создание UI элементов не исключение, можно отбросить лишнее, все будет прекрасно работать.

Reply:
не работает у меня